terminus-ssh/package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "build": "webpack --progress --color",
     "watch": "webpack --progress --color --watch",
-    "postinstall": "xcopy /i node_modules\\ssh2\\util\\pagent.exe util\\"
+    "postinstall": "xcopy /i /y node_modules\\ssh2\\util\\pagent.exe util\\"
```

Comment:
Fixed because when I executed a command `node scripts\build-native.js` in windows, it stopped responding to xcopy